libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -1,49 +1,22 @@
 <form [formGroup]="searchForm" (submit)="searchBooks()">
-  <mat-form-field floatLabel="never">
+  <mat-form-field>
+    <mat-label id="search-label" aria-label="search-label">Search</mat-label>
```

Comment:
Why would a `mat-label` need an `aria-label`?

Reply:
removing it since it is redundant here